### PR TITLE
🐛 PRTL-838 tooltip positioning off in IE

### PR DESCRIPTION
**Pull request by cheapsteak**
moveTooltip needs to wait to be called after tooltip's offsetHeight has the correct value

### CODE REVIEW

File: src/packages/@ncigdc/uikit/Tooltip/GlobalTooltip.js [hunk 1]
```diff
@@ -11,12 +12,16 @@ class GlobalTooltip extends Component {
...
-  moveTooltip = event => {
-    this.globalTooltip.style.left = `${event.pageX}px`;
-    this.globalTooltip.style.top = `${event.pageY -
-      this.globalTooltip.offsetHeight -
-      15}px`;
-  };
+  moveTooltip = _.throttle(event => {
+    const left = event.pageX;
+    const top = event.pageY;
+    setTimeout(() => {
```
Comment (Jephuff) [inline src/packages/@ncigdc/uikit/Tooltip/GlobalTooltip.js]: Does this fix it, or just make the edge case edgier? Would it make sense to add a componentDidUpdate and add/remove the listener based on `props.tooltip.Component`?

Comment (cheapsteak) [inline src/packages/@ncigdc/uikit/Tooltip/GlobalTooltip.js]: This either fixes it or makes it edgier to the point of being undifferentiable from being fixed :p

I'm not sure if the position would be correct with the current global tooltip solution if the listener is added after componentDidUpdate. The mouse move event may happen before that lifecycle hook is called

Comment (Jephuff) [inline src/packages/@ncigdc/uikit/Tooltip/GlobalTooltip.js]: yeah, you are right.
Wanna switch this from using top/left to using transforms well your in here? :D

Comment (cheapsteak) [inline src/packages/@ncigdc/uikit/Tooltip/GlobalTooltip.js]: Worried about potential bugs  
This tooltip has position absolute so it wouldn't cause any DOM reflows  
Would recommend we prioritize minimizing bugs unless this proves to be a performance bottleneck

Comment (Jephuff) [inline src/packages/@ncigdc/uikit/Tooltip/GlobalTooltip.js]: Fair. Realized we also don't need the height to calculate tooltip position, but that would be a bit more of a rewrite, so maybe in the future we can do that :)

**Pull request merged**